bridge/client/app/_utils/form.utils.ts

```diff
@@ -21,6 +21,19 @@ export class FormUtils {
     return null;
   }
 
+  public static isUrlOrSecretValidator(
+    control: AbstractControl
+  ): { urlOrSecret: { value: boolean } } | { space: { value: boolean } } | null {
+    if (control.value) {
+      if ((control.value.search(/^http(s?):\/\//) === -1) && (control.value.search(/^{{(.+)}}/) === -1)) {
```

Comment:
I think that in case of a secret (and also variable from the payload), it always has to start with a dot followed by some characters, so we could also add a check for the dot here?
```suggestion
      if ((control.value.search(/^http(s?):\/\//) === -1) && (control.value.search(/^{{(\..+)}}/) === -1)) {
```